src/functions.js

```diff
@@ -4,7 +4,7 @@
  * @param {string} selector
```

Comment:
Still have it in my head that this isn't supported yet but that's because some of the plugins in Altis have old build scripts and tooling now. This makes sense to me 👍